orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -72,6 +73,19 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
   }
 
   @Override
+  long getExponentialBackoffPeriod(Duration taskDuration) {
+    if (taskDuration.toMillis() > TimeUnit.MINUTES.toMillis(60)) {
+      // task has been running > 60min, drop retry interval to every 2 minutes
+      return TimeUnit.SECONDS.toMillis(120)
```

Comment:
maybe `Math.max(backoffPeriod, TimeUnit.SECONDS.toMillis(120))` here (and below)